test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -800,6 +802,386 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 			}
 		})
 
+		It("[test_id:50964] Offline Higher CPUID's", func() {
+			var offlineCPUs, reservedCpus, isolatedCpus string = "", "", ""
+			//This map is of the form numaNode[core][cpu-siblings]
+			var numaCoreSiblings map[int]map[int][]int
+			var onlineCPUInt int
+			for _, node := range workerRTNodes {
+				onlineCPUCount, err := nodes.ExecCommandOnNode([]string{"nproc", "--all"}, &node)
+				Expect(err).ToNot(HaveOccurred())
+				onlineCPUInt, err = strconv.Atoi(onlineCPUCount)
+				Expect(err).ToNot(HaveOccurred())
+			}
+			//This test requires at least more than 5 cpus:
```

Comment:
To be consistent with https://github.com/openshift/cluster-node-tuning-operator/pull/470 should be at least 8

Also check it in every test